### DISCUSSION TITLE
Allow Admins to Change User's User Group After Creation

### DISCUSSION THREAD

**Discussion by eroc1990** [mealie-recipes/mealie#4540] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Once a user has been created, it's not possible for an admin to move their home group to another user group.

### Please provide a concise description of the feature that would resolve your issue.

The ability to be able to move the home user group (which I assume would require on-the-fly reassignment of their recipes) would allow me to organize users' memberships after user creation.

### Please consider and list out some caveats or tradeoffs made in your design decision

?

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Currently there are a lot of issues with moving users between groups, which is why this currently isn't supported. We suggest that if you create a user in the wrong group then you should delete the user and create them in the correct group. Keep in mind that groups are not the same as households, and users can be freely moved between households.

There's an [open PR](https://github.com/mealie-recipes/mealie/pull/4252) almost ready to be merged which allows admins to create invites for users for other groups/households.

**Comment by eroc1990** [comment]
I can imagine that would be difficult, logistically. It'd be awesome to see that in the future, though, in the event pre-configured users (like the two I currently have and the more I plan on having) should actually be members of one of the other created groups.

The group/household specific invites will be a nice way to have people configured to the correct group from the start, and I look forward to that being merged, but if it's possible I do think this kind of control should be implemented some time down the line. The one user I need to move holds the bulk of the recipes uploaded, and it would be a pain to have to reassociate them again, since I already went through it once rebuilding my instance after ditching my pre 1.0 setup for the current branch.
